.github/workflows/build_and_deploy_dev.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and deploy to dev
+on:
+  workflow_dispatch:
+    inputs:
+      config-file-name:
+        description: 'Config-fil (uten .json) som skal deployes'
+        required: true
+        default: 'dev-sbs-intern'
+      cluster:
+        description: 'Cluster å deploye til (dev-sbs, labs-gcp eller dev-gcp)'
+        required: true
+        default: 'dev-sbs'
+
+# FIXME: Legg til docker layer caching, f.eks. https://github.com/marketplace/actions/build-docker-images-using-cache
+jobs:
+  build:
+    name: Build and deploy to dev
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE: docker.pkg.github.com/${{ github.repository }}/${{ github.event.repository.name }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          cache: 'npm'
+      - name: Copy env-file for dev
+        run: cp nais/dev/${{ github.event.inputs.config-file-name }}.env .env.local
+      - name: Install dependencies
+        run: npm ci
+      - name: Run tests
+        run: npm run test
+      - name: Build application
+        run: npm run build
+      - name: Login to GitHub package registry
+        run: |
+          echo ${GITHUB_TOKEN} | docker login -u ${GITHUB_REPOSITORY} --password-stdin docker.pkg.github.com
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create artifact version
+        id: artifact-version
+        uses: navikt/sosialhjelp-ci/actions/create-artifact-version@master
+      - name: Get complete tag
+        run: |
+          echo "COMPLETE_TAG=${{ github.event.inputs.config-file-name }}-$( date +%s )-${{ steps.artifact-version.outputs.version }}" >> $GITHUB_ENV
```

Comment:
blir image tag her eks: `prod-20211130-<steps.artifact-version.outputs.version>`?
Trodde `steps.artifact-version.outputs.version` allerede inneholdt dato - https://github.com/navikt/sosialhjelp-soknad/releases/

Reply:
Det blir mer på formatet `dev-sbs-TIMESTAMP-<steps.artifact-version.outputs.version>`. For veiviserappene la vi på timestamp for å sikre at gjentagende bygg og deploy ikke ville generere like tags.

Reply:
skjønner 👍